src/main/kotlin/uk/gov/justice/digital/hmpps/pecs/jpc/domain/move/Event.kt

```diff
@@ -46,8 +52,9 @@ data class Event constructor(
   @Column(name = "eventable_id")
   val eventableId: String,
 
-  @Transient
-  val details: Map<String, Any>?,
+  @Convert(converter = DetailsConverter::class)
```

Comment:
So event details (i.e. reg details) were never persisted.  I am not going to worry about backfilling these for now.  I think we are probably fine to say going forwards we will be using the details from the events.